contracts/src/CrossChainApplications/README.md

```diff
@@ -90,11 +85,7 @@ function receiveTeleporterMessage(
 ) external {
 ```
 
-Now that the function declarations are set up, we can move on to implementation.
-
-> To incentivize relayers to deliver our cross chain messages from source to destination, we will need to support `ERC20` fees. Relayer fees aren't needed when running your own relayers to deliver your own messages and pay costs out of pocket.
```

Comment:
I liked a note to explain the use of fees before the step of importing the IERC20 contract, and calling out that fees aren't needed if relaying own message.